src/overlords/core/scout.ts

```diff
@@ -29,6 +29,8 @@ export class ScoutOverlord extends Overlord {
 	private handleScout(scout: Zerg): void {
 		if (!scout.pos.inRangeTo(this.pos, 3) && !scout.pos.isEdge) {
```

Comment:
this bug was fixed in a recent commit; update from master and use that version of this file (which has been moved as well)